testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -447,8 +447,7 @@ impl EventDispatcher {
                         json!({
                             "recipient": reward.address.to_string(),
                             "coinbase_amount": reward.coinbase.to_string(),
-                            "tx_fees_anchored_shared": reward.tx_fees_anchored_shared.to_string(),
-                            "tx_fees_anchored_exclusive": reward.tx_fees_anchored_exclusive.to_string(),
+                            "tx_fees_anchored": reward.tx_fees_anchored.to_string(),
```

Comment:
@zone117x -- this is a change to one of the event types (a simplification because there's never any "shared" anchored tx fees.

@jcnelson -- can you update the example in the `docs/event-dispatcher.md` file?

Reply:
Sure thing!

Reply:
Addressed in ee12847